test/extensions/clusters/dynamic_forward_proxy/cluster_test.cc

```diff
@@ -172,6 +183,14 @@ TEST_F(ClusterTest, InvalidLbContext) {
   EXPECT_EQ(nullptr, lb_->chooseHost(nullptr));
 }
 
+// Verify cluster attaches to a populated cache.
```

Comment:
Do we have any tests which mix and match some hosts cached and some not? I'm pretty confident it works to have some pre-populated, add some, and they'd get pre-populated for a new cluster, but I think it may be worth covering.  If you think it's redundant with the existing cache tests I'll take your word for it :-)

Reply:
I spent a little time looking at whether I can add a useful test in this area, and unfortunately I don't think I can. The cluster test uses a mock cache so it would be synthetic. The cache test is independent, and the integration test is unfortunately forced to use localhost only so there is no good way to get multiple entries in. 

I think I'm going to go ahead and merge this and will think if there is some other type of test I can add that is useful. If you have any ideas let me know and I will do it as a follow up.